model/cpu/metrics.yaml

```
@@ -0,0 +1,35 @@
+groups:
+  # cpu.* metrics
+  - id: metric.cpu.time
+    type: metric
+    metric_name: cpu.time
+    stability: development
+    brief: "Seconds each logical CPU spent on each mode"
+    instrument: counter
+    unit: "s"
+    attributes:
+      - ref: cpu.mode
+        note: "Following states SHOULD be used: `user`, `system`, `nice`, `idle`, `iowait`, `interrupt`, `steal`"
+      - ref: cpu.logical_number
+
+  - id: metric.cpu.utilization
+    type: metric
+    metric_name: cpu.utilization
+    stability: development
+    brief: "For each logical CPU, the utilization is calculated as the change in cumulative CPU time (cpu.time) over a measurement interval, divided by the elapsed time."
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: cpu.mode
+        note: "Following modes SHOULD be used: `user`, `system`, `nice`, `idle`, `iowait`, `interrupt`, `steal`"
+      - ref: cpu.logical_number
+
+  - id: metric.cpu.frequency
+    type: metric
+    metric_name: cpu.frequency
+    stability: development
+    brief: "Operating frequency of the logical CPU in Hertz."
+    instrument: gauge
+    unit: "{Hz}"
```

nit: `Hz` is defined in [UCUM](https://unitsofmeasure.org/ucum#para-30)


```suggestion
    unit: "Hz"
```

---

fix and regenerated docs in https://github.com/open-telemetry/semantic-conventions/pull/1896/commits/ca0ab62e3e80bb9f33ec3f9ad367a054033c85c5

thank you!